.github/workflows/github-actions-test.yml

```
@@ -29,7 +34,14 @@ jobs:
       # Plain Maven from the repository root: the reactor satisfies the
       # pic-sure-bom import, so no bootstrap, token, or warm cache is needed.
       - name: Verify with Maven
-        run: mvn -T1C -B verify
+        run: mvn -T1C -B verify -Dmaven.test.failure.ignore=true
+        with:
```

actionlint (1.7.12)

[error] unexpected key "with" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)